### DISCUSSION TITLE
React is included in the dev dependencies of the design system starter template in the UI workspace

### DISCUSSION THREAD

**Discussion by ghdtjgus76** [vercel/turborepo#8156] [Ideas]
### Goals

1. React is included in the dev dependencies of the design system starter template in the UI workspace
<img width="352" alt="image" src="https://github.com/vercel/turbo/assets/81316050/fe55c381-74c2-4d90-b612-e0281afea69d">

### Background

I was planning to use the design system starter template from TurboRepo. 
However, while examining the package.json file in the UI workspace, I noticed that React is listed under devDependencies. 
As far as I know, React should be specified as a dependency, not a devDependency. 
If there's anything wrong with this, I would greatly appreciate it if you could let me know.

### Proposal

React should be specified as a dependency, not a devDependency.

**Comment by anthonyshew** [comment]
Happy to take a PR for this!

**Comment by anthonyshew** [comment]
Looks like this was fixed! https://github.com/vercel/turbo/commit/ca758d962d212aafb14f521228db48f686212f41
